ipa-multipoint/ipa_multipoint_jni/src/lib.rs

```diff
@@ -82,16 +80,20 @@ pub extern "system" fn Java_org_hyperledger_besu_nativelib_ipamultipoint_LibIpaM
         .convert_byte_array(input)
         .expect("Cannot convert jbyteArray to rust array");
 
-    let committer = &CONFIG.committer;
-
-    let commitment = ffi_interface::commit_to_scalars(committer, &input).unwrap();
+    let commitment = ffi_interface::commit_to_scalars(&CONFIG, &input).unwrap();
 
     env.byte_array_from_slice(&commitment)
         .expect("Couldn't convert to byte array")
 }
 
-/// Commit_root receives a list of 32 byte scalars and returns a 32 byte commitment.to_bytes()
-/// This is ported from rust-verkle.
+/// Commit_root receives a list of 32 byte scalars and returns a 32 byte array
+///
+/// The 32 bytes represents groupToField(commitment).to_bytes()
+///
+/// TODO: We can remove this method as its the same as calling commit and then groupToField
+/// TODO: This has not been done because it is a breaking API change, that changes the bindings.
+/// TODO: Once downstream besu-verkle switches to using commit and then groupToField, making this
+/// TODO: method unused, then we can remove it.
 #[no_mangle]
 pub extern "system" fn Java_org_hyperledger_besu_nativelib_ipamultipoint_LibIpaMultipoint_commitRoot(
```

Comment:
This means that besu-verkle does not need to be modified to make it match the other implementations.